core/src/main/scala/delorean/package.scala

```diff
@@ -39,12 +39,12 @@ package object delorean {
     def unsafeToFuture(): Future[A] = {
       val p = Promise[A]()
 
-      self runAsync {
+      self unsafePerformAsync {
```

Comment:
This nuance might also need a similar strategy as to what we're doing in Knobs: https://github.com/Verizon/knobs/blob/master/project/CrossLibraryPlugin.scala#L22 - including a different source path based on the selected scalaz-stream version. In this case, it would be the scalaz version.